src/contexts/globalLayout/GlobalNavbar/CollectionSaveButtonWithCaption.tsx

```diff
@@ -0,0 +1,155 @@
+      clearTimeout(deactivateHoverCardTimeoutRef.current);
+    }
+    setIsShowPopup(true);
+  }, []);
+
+  const handleCloseCaption = useCallback(() => {
+    deactivateHoverCardTimeoutRef.current = setTimeout(
+      () => setIsShowPopup(false),
+      ANIMATED_COMPONENT_TRANSITION_MS
+    );
+  }, []);
+
+  const handleCaptionChange = useCallback((event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setCaption(event.target.value);
+  }, []);
+
+  const handleSubmit = useCallback(async () => {
+    setIsLoading(true);
+    await onSave(caption);
+    handleCloseCaption();
+    setIsLoading(false);
+  }, [caption, handleCloseCaption, onSave]);
+
+  const isDisabledonSave = useMemo(() => {
+    return caption.length > 600;
+  }, [caption]);
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  return (
+    <StyledConfirmationContainer>
+      <Button onClick={isShowPopup ? handleCloseCaption : handleOpenCaption} disabled={disabled}>
+        <HStack gap={4} align="center">
+          {label}
+          <StyledChevronSvg
+            isActive={isShowPopup}
+            width="12"
+            height="7"
+            viewBox="0 0 12 7"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path d="M1.33337 1L6.00004 5.66667L10.6667 1" stroke="#FEFEFE" strokeMiterlimit="10" />
+          </StyledChevronSvg>
+        </HStack>
+      </Button>
+      <StyledCardContainer gap={hasUnsavedChange ? 12 : 24} isActive={isShowPopup}>
+        {isShowPopup && (
+          <>
+            <HStack justify="flex-end">
+              <StyledCloseButton onClick={handleCloseCaption}>
+                <CloseIcon isActive={true} />
+              </StyledCloseButton>
+            </HStack>
+            {hasUnsavedChange ? (
+              <>
+                <StyledConfirmationContent gap={8}>
+                  <BaseM>Share your update to the feed with an optional note.</BaseM>
+                  <TextAreaWithCharCount
+                    currentCharCount={caption.length}
+                    maxCharCount={600}
+                    onChange={handleCaptionChange}
+                    hasPadding
+                    placeholder="Add a note..."
+                    textAreaHeight="50px"
+                  />
+                </StyledConfirmationContent>
+
+                <Button onClick={handleSubmit} disabled={isDisabledonSave} pending={isLoading}>
+                  Save
+                </Button>
+              </>
+            ) : (
+              <StyledNoChangesTitle>No changes made yet.</StyledNoChangesTitle>
+            )}
+          </>
+        )}
+      </StyledCardContainer>
+    </StyledConfirmationContainer>
+  );
+}
+
+const StyledConfirmationContainer = styled(VStack)`
+  position: relative;
+`;
+
+const StyledChevronSvg = styled.svg<{ isActive: boolean }>`
+  transform: ${({ isActive }) => (isActive ? 'rotate(180deg)' : 'rotate(0deg)')};
+  transition: transform 0.2s ease-in-out;
```

Comment:
i love this animation but i don't think it's quite on-brand 😅 mind if we remove it?

Reply:
Definitely! I've removed it